hazelcast-sql/src/test/java/com/hazelcast/sql/impl/schema/IMapResolverTest.java

```diff
@@ -20,20 +20,13 @@
 import com.hazelcast.jet.sql.SqlTestSupport;
 import com.hazelcast.jet.sql.impl.schema.model.Person;
 import com.hazelcast.sql.impl.client.SqlClientService;
-import com.hazelcast.test.HazelcastParallelClassRunner;
-import com.hazelcast.test.annotation.ParallelJVMTest;
-import com.hazelcast.test.annotation.QuickTest;
 import org.junit.BeforeClass;
 import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
 
 import java.math.BigDecimal;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@RunWith(HazelcastParallelClassRunner.class)
-@Category({QuickTest.class, ParallelJVMTest.class})
```

Comment:
We should have at least `@Category(QuickTest.class)` here. It is used for out PR builder and test coverage reports.

Reply:
Also we have `HazelcastSerialClassRunner`, so, you can safely replace `HazelcastParallelClassRunner`

Reply:
`@Category` is inherited, so the PR builder will pick it up.